pkg/minikube/constants/constants.go

```diff
@@ -85,3 +85,13 @@ func GetMinishiftHomeDir() string {
 		return filepath.Join(util.HomeDir(), ".minishift")
 	}
 }
+
+// ProfileAuthorizedKeysPath returns the path of authorized_keys file in profile dir used for authentication purpose
```

Comment:
Why does this have to be in the minikube package?
And these, like previous one's are surely not constant at all...

Can we not have a separate `minishift`-like profile variables function in our own minishift namespace?